content/stratisd-release-notes-3.5.2.md

```diff
@@ -0,0 +1,42 @@
++++
+title = "stratisd 3.5.2 Release Notes"
+date = 2023-03-17
+weight = 31
+template = "page.html"
+render = true
++++
+
+*mulhern, Stratis Team*
+
+`stratisd` 3.5.2 includes three significant enhancements as well as a bug
+fix.
+
+The enhancements are:
+
+* `stratisd` 3.5.2 is the first `stratisd` release to include a subpackage,
+`stratisd-tools`, which incorporates `stratis-dumpmetadata`, an application
+which may be used for troubleshooting.
+* `stratisd` 3.5.2 now depends on `devicemapper-rs` 0.33.1, which includes
+support for synchronization between udev and devicemapper[stratisd pr 3069]. See
+the [devicemapper-rs] changelog for additional details.
+* `stratisd` 3.5.2 modifies the way takeover by `stratisd` from `stratisd-min`
```

Comment:
We can if you think it will be useful. I left out any mention of NFS because the change itself is not NFS-specific. But if you want to throw in a one sentence motivator for the change, I'll be happy to pull it in.